source/globals.d.ts

```diff
@@ -2,15 +2,13 @@
 
 type AnyObject = Record<string, any>;
 
-type FeatureID = 'use the __filebasename variable';
+type FeatureID = string;
```

Comment:
We could use an opaque type here, to enforce better semantics.

Reply:
That's what this was. It should be undone

```suggestion
type FeatureID = 'use the __filebasename variable';
```